### ISSUE TITLE
General Data Protection Regulation in Qualification Procedure and RIPS

### ISSUE THREAD

**Issue by kyordhel** [RoboCupAtHome/RuleBook#450]
New regulations regarding user data protection might cause troubles for the diffusion of what happens in RoboCup@Home.

During the Qualification Procedure, team candidates send Qualification Materials for inspection. These materials are inspected by the OC, but also are meant to be made public for further reference. However, uploading the videos might void certain regulations. Therefore, my proposal is update the Rrulebook an CFP requesting the team leader to provide a signed copy of a disclose agreement, in which:
- All people appearing in the TDP and provided multimedia materials grant us permission to upload, copy, project, and redistribute the material.
- Team authorizes us to use the TDP and provided multimedia materials in whatever ends are convenient to promote and advertise the league.
- (Ideas?)

The same should be issued along with the registration form, in which team members allow us to take video, photos, and record their voices and publish such data for promotion and advertising, as well as for research purposes under an [open access] licence.

**Comment by justinhart** [comment]
A typical IRB consent form indicates that participation is voluntary and
that consent can be revoked at any time, and has a data management plan. I
like the basic idea, but we'll need to tweak it a bit to conform to these
sorts of protections. Those apply in US & Canada, not sure about EU.

**Comment by moriarty** [comment]
I thought when you participated in RoboCup you arrive on sight and sign away your rights to be recorded as soon as you arrive and pick up your registration package...

Maxon Motors got a hold of this photo which was taken by RoboCup 2013: https://www.flickr.com/photos/robocup2013/9156473611/ in Einthoven  

![13139274_10207805446208520_8286661214773194580_n](https://user-images.githubusercontent.com/55676/40154771-f3a294d4-5944-11e8-892c-19822fc56d32.jpg)

A few years later I heard about the photo from someone who attended Hannover Messe and saw it, but hadn't seen it until I was randomly sitting on a train and the person across from me was reading an Industry 4.0 magazine with it on the back cover...

Long story short:
Don't all participants/visitors to robocup agree to some data waiver which covers these cases?
If not, I should have been informed about that photo...

But also, isn't this a general issue for almost all leagues in RoboCup? so shouldn't the RoboCup update their existing consent form?

**Comment by kyordhel** [comment]
@moriarty AFAIK participants must sign that form, but I am not so sure about the general public attending to the venue.

The case of @Home is slightly more critical because, unlike other leagues, it doesn't involves some pictures and videos for advertising. We are aiming to record real user-robot interaction for research purposes, and that includes:
1. lifetime use (you can't remove people's data from a public dataset).
2. free, unrestricted distribution.

As @justinhart pointed out, we need more data.

**Comment by warp1337** [comment]
FYI, concerning Europe: https://www.eugdpr.org/key-changes.html
